ibis/common/collections.py

```diff
@@ -150,6 +150,16 @@ def __eq__(self, other):
         return dict(self.items()) == dict(other.items())
 
 
+class ConflictingValuesError(ValueError):
```

Comment:
Can you put this in our `exceptions` module? Sorry, Jack Diedrich might not approve 😂